Autopilot/AttitudeManager/Multirotor.c

```diff
@@ -42,10 +42,10 @@ void armVehicle(int delayTime){
     asm("CLRWDT");
     Delay(delayTime);
```

Comment:
we should probably put the clear watchdog assembly isntruction within the delay function. Up to you if you want to do it in this PR, it'll clean up this code however